plugins/tracker-resources/src/components/issues/KanbanView.svelte

```diff
@@ -245,8 +245,7 @@
       return
     }
     return {
-      [groupByKey]: groupValue,
-      space: doc.space
```

Comment:
We need to check if space update is not required. You could try diffUpdate

Reply:
@haiodo which case it can be required?

Reply:
@haiodo I couldn't find a case where we need to update `space`, so I decided that we can just get rid of it 

If I use `diffUpdate` in generic `Kanban`, this also work, thanks

Reply:
It could be in case of all issues, and move between spaces occur, but I've not sure how we handle it here.

Reply:
So probable we need to add one more issue to check it

Reply:
Now it will be broken, because diffUpdate skip space changes...

Reply:
heh